javalib/src/main/scala/java/io/InputStreamReader.scala

```diff
@@ -91,6 +91,8 @@ class InputStreamReader(private[this] var in: InputStream,
         readMoreThroughOutBuf(cbuf, off, len)
       }
     }
+    else
+      -1
```

Comment:
Sorry I used bits of this file as styling reference instead of CODINGSTYLE.md. Though I don't understand why this is OK (l.149) when the above is not :/
```
if (decoder.flush(out).isOverflow)
  InputStreamReader.Overflow
else {
  ...
}
```